spec/service_spec.rb

```diff
@@ -48,6 +48,49 @@
     expect { service.resource_get('api') }.to raise_error(Exception, "BLAH")
   end
 
+  context "#get_resource_by_ems_ref" do
+    it "fetches data_center" do
```

Comment:
It looks like we are using two different version. In my case I am using 3.6 and you are using 3.5. I will add a test using response you got.